### DISCUSSION TITLE
How to run ci moved designs list

### DISCUSSION THREAD

**Discussion by vijayank88** [The-OpenROAD-Project/OpenLane#1903] [General] [answered]
With earlier setup I used to run built-in examples with following command,
```
make mount
./flow.tcl -design aes
```
With latest commit `aes` moved inside `ci/aes`. How do I run this with `flow.tcl` now for `aes`?

**Comment by kareefardi** [comment, marked as answer]
```
./flow.tcl -design ./ci/aes
```
